### ISSUE TITLE
Why does my `overlayTop` tab bar appear at bottom?

### ISSUE THREAD

**Issue by boringer** [GeekyAnts/NativeBase#1330]
## react-native, react and native-base version

React Native: 0.48.4.

React: 16.0.0-alpha.12.

Native Base: 2.3.3.

## Expected behaviour

At top.

## Actual behaviour

At bottom.

## Steps to reproduce (code snippet or screenshot)

```jsx
<Container style={{paddingVertical: 50}}>
    <Tabs tabBarPosition="overlayTop">
        <Tab heading="Tab 1">
            <Text>1</Text>
        </Tab>
        <Tab heading="Tab 2">
            <Text>2</Text>
        </Tab>
        <Tab heading="Tab 3">
            <Text>3</Text>
        </Tab>
    </Tabs>
</Container>
```

## Screenshot of emulator/device

![img_6a5eca6c4867-1](https://user-images.githubusercontent.com/14685751/32135126-3a1b81d4-bc2c-11e7-8ac3-18fc723026cd.jpeg)

## Is the bug present in both ios and android or in any one of them?

Only tested on iOS.

**Comment by akhil-geekyants** [comment]
Fixed position in #1355

**Comment by SupriyaKalghatgi** [comment]
Fixed with NativeBase [2.3.5](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.5)
